src/client/pfs/pfs.proto

```diff
@@ -340,23 +291,27 @@ service InternalAPI {
 
   // Commit rpcs
   // StartCommit creates a new write commit from a parent commit.
-  rpc StartCommit(StartCommitRequest) returns (google.protobuf.Empty) {}
+  rpc StartCommit(StartCommitRequest) returns (Commit) {}
+  // Fork creates a commit on a new branch.
+  rpc Fork(ForkRequest) returns (Commit) {}
```

Comment:
I'm not sure if `CreateBranch` captures the fact that a new commit is being created.  Further, `Fork` always takes a parent commit, whereas `CreateBranch` sounds like it can create an entirely new branch, which is actually what `StartCommit` does.

How about `ForkCommit`?